eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -99,7 +112,7 @@ jobs:
 
   # Synchronize new content in the VMR during PRs (we expect this to come
   - ${{ if and(not(parameters.isBuiltFromVmr), eq(variables['Build.Reason'], 'PullRequest')) }}:
-    - template: ../steps/vmr-pull-updates.yml
+    - template: /src/installer/eng/pipelines/templates/steps/vmr-pull-updates.yml@self
```

Comment:
I don't think we can have absolute paths because we build from VMR and installer. I tried this and relative paths work as well.